File: 1941-check-if-all-characters-have-equal-number-of-occurrences/1941-check-if-all-characters-have-equal-number-of-occurrences.py
class Solution:
    def areOccurrencesEqual(self, s: str) -> bool:
        st=set()
        dic=dict()
        for i in s:
            if dic.get(i)==None:
                dic[i]=1
            else:
                dic[i]+=1
        for key,value in dic.items():
            # print(value,end=' ')
            st.add(value)
        if len(st)>1:
            return False
        return True

File: 1941-check-if-all-characters-have-equal-number-of-occurrences/1941-check-if-all-characters-have-equal-number-of-occurrences.cpp
class Solution {
public:
    bool areOccurrencesEqual(string s) {
        map<char , int> m;
        for(auto i:s)
        {
            m[i]++;
        }
        int temp=m[s[0]];    
        for(auto i:m)
        {  

            if(temp!=i.second)
            return false;
        }
        return true;
    }
};